ROMFS/CMakeLists.txt

```diff
@@ -4,6 +4,22 @@ message(STATUS "ROMFS: ${config_romfs_root}")
 set(romfs_temp_dir ${PX4_BINARY_DIR}/ROMFS/${config_romfs_root})
 set(romfs_src_dir ${PX4_SOURCE_DIR}/ROMFS/${config_romfs_root})
 
+# Copy the optional files into ROMFS
+message(STATUS "The optional files: ${romfs_optional_files}")
```

Comment:
Minor, but could you have this print only when optional files are being included? I'm trying to keep the output as simple as possible so people actually look at it.

Reply:
Ok. Will fix. See next version that I push.